qt/aqt/addons.py

```diff
@@ -816,30 +825,28 @@ def _onAddonItemSelected(self, row_int: int) -> None:
         gui_hooks.addons_dialog_did_change_selected_addon(self, addon)
         return
 
-    def selectedAddons(self) -> List[str]:
+    def selectedAddons(self) -> List[AddonMeta]:
```

Comment:
There are some add-ons out there that call this function and will break if the signature changes. I recommend you make a new method called selected_addon_meta() instead, as that should avoid breaking things, and you won't have to change as many things in this PR.

Reply:
Done.

Reply:
Nice and clean, thank you :-)